website/docs/reference/opspec/op-directory/op/parameter/object.md

```diff
@@ -18,3 +18,33 @@ An object to use as the value of the parameter when no argument is provided.
 
 ### isSecret
 A boolean indicating if the value of the parameter is secret. This will cause it to be hidden in UI's for example. 
+
+## Example
+The op uses an object input, and echos the values of the object's keys.
+
+```yaml
+inputs:
+    example-input:
+        object:
+            default:
+                keyA: "valueA"
+                keyB: ["subValueB1", "subValueB2", "subValueB3"]
+                keyC:
+                    subKeyC1: "subValueC1"
+                    subKeyC2: "subKeyC2"
+run:
+    container:
+        image: { ref: 'apteno/alpine-jq' }
```

Comment:
might consider using straight alpine

Reply:
yeah, I was initially thinking I would get valid json out of the object and that I could format it, but that is not the case

Reply:
objects/arrays do get serialized to JSON but if you're injecting it in a shell you'll need to make sure to escape it since quotes have special meaning to shells.

Reply:
@gunnarg-remitly  any update here?